Schmidt/template/README.md

```diff
@@ -1,13 +1,14 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+
+Factorial is a function that returns the factorial of a number, defined as that number times every integer less than it down to 1. factorial(10) returns 10!
 
     >>> from simple_functions import factorial
     >>> factorial(10)
-    9
+    3628800
 
-and this other part does something.  I forget why that I did it:
+The function fibonnacci returns a fibonacci sequence of all numbers that are less than the given numbers. 
 
     >>> fibonnaccci(100)
     [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
 
-If you can't use it, its kind of your problem, not mine!
+Readme updated 27.08.2019 by Sarah Jane Schmidt
```

Comment:
@sjschmidt - one minor comment here: in general on git it's not a good idea to use "dated" information like this - the history of updates of a file are always available via `git log README.md`, so this is not necessary, and text like this has a tendancy to get out-of-date (i.e., if someone *other* than you edited this readme, they might not think to add to this file).  So it's best to just leave this sort of information off from anything tracked in git unless there's a specific known need for it.